### PR TITLE
Configadd

### PR DESCRIPTION
**Pull request by yakyakyakyak**
Added config, minor structure additions

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -9,8 +9,8 @@ org.gradle.jvmargs=-Xmx1G
 
 # Mod Properties
 	mod_version = 1.0.0
-	maven_group = com.example
-	archives_base_name = fabric-example-mod
+	maven_group = com.refsonyak.endgameaddons
+	archives_base_name = endgame-addons-mod
```
Comment (YTG1234) [inline gradle.properties:13]: Absolutely not.

File: src/main/java/net/refsonyak/endgameaddons/Config.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+public class Config {
+    /*Class that contains all configurable variables for quick'n'easy balance changes
```
Comment (YTG1234) [inline src/main/java/net/refsonyak/endgameaddons/Config.java:2]: Please word more carefully and format the comment to either use `//` or have `/*` and `*/` on a separate line.

File: src/main/java/net/refsonyak/endgameaddons/Config.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+      To use, simply create a class in here, give it some variables, import it into your
+      file. Call on it using Config.<yourclass>.<yourvariable> and voila!*/
+
+    public class dropRates {
```
Comment (YTG1234) [inline src/main/java/net/refsonyak/endgameaddons/Config.java:6]: This is horrible naming conventions and classes should be `static`. You also do not need inner classes for everything. Static fields should be enough. Please also provide information about de/serialization and configuration screens.

**Review by YTG1234 — CHANGES_REQUESTED** [review]
Do not delete the license and readme files. What are you thinking?

**Comment by Juuxel** [comment]
@YTG1234 I'm pretty sure this PR isn't supposed to be here. (See #80 which was closed.)

**Comment by yakyakyakyak** [comment]
hi i fked up, thought this was my fork. sorry i'm garbage

**Pull request closed without merging**